laravel-app/app/Http/Controllers/ProfileController.php

```diff
@@ -6,39 +6,64 @@
 use App\Models\User;
 use App\Services\ProfileService;
 use Illuminate\Database\QueryException;
-use Illuminate\Support\Facades\DB;
+use Illuminate\View\View;
+use Illuminate\Http\RedirectResponse;
+
 class ProfileController extends Controller
 {
-    protected $userService;
+    protected ProfileService $userService;
 
+    /**
+     * Create a new controller instance.
+     *
+     * @param ProfileService $userService
+     * @return void
+     */
     public function __construct(ProfileService $userService)
     {
         $this->userService = $userService;
     }
 
-    public function index()
+    /**
+     * Show the profile page.
+     *
+     * @return View
+     */
+    public function index(): View
     {
         $user = auth()->user();
         return view('profile.index', compact('user'));
     }
 
-    public function edit($profile)
+    /**
+     * Show the profile edit page.
+     *
+     * @param User $profile
+     * @return View
+     */
+    public function edit($profile): View
```

Comment:
Since they are using Laravel 10. Laravel already removed doc block and used return type declaration.

Docs:
https://laravel.com/docs/10.x/releases#types
https://www.php.net/manual/en/language.types.declarations.php

Demo:
https://www.reddit.com/r/laravel/comments/11328e0/quick_demo_of_the_laravel_10x_shift/